main.go

```diff
@@ -629,6 +629,20 @@ func main() {
 	if err != nil {
 		l.Panicf("Server problem: %+v", err)
 	}
+	updateSignal := make(chan os.Signal, 1)
+	signal.Notify(updateSignal, unix.SIGHUP)
+	go func() {
+		for {
+			s := <-updateSignal
+			l.Infof("Got %s signal: reloading configuration", s.String())
```

Comment:
Please use code similar to existing handler (where we use `unix.SignalName`, etc)